### DISCUSSION TITLE
Welcome to linkchecker Discussions!

### DISCUSSION THREAD

**Discussion by anarcat** [linkchecker/linkchecker#655] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together .

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anarcat** [comment]
so yeah i just did this, not sure if i followed process, but it seemed harmless enough to enable this, and i figured it was a useful place to have, especially since i needed some place to do an announcement (and not this one).

**Reply by cjmayo** [reply]
Good idea! Announcements are something different to issues - no need to manage closing them.

**Comment by jayavenkats** [comment]
when executing the linkchecker, getting error SSL: Certificate verify failed. any suggestions to overcome this error or workaround?

**Reply by cjmayo** [reply]
There is a `sslverify` setting that can be used in the linkcheckerrc configuration file:
https://linkchecker.github.io/linkchecker/man/linkcheckerrc.html

Info about where to find the linkcheckerrc at the top of that page.

(Do start new discussions when raising a new topic. It may help other people).

**Reply by jayavenkats** [reply]
thanks for the reply.  do i have to give this sslverify=0 in command line during execution ?

**Reply by cjmayo** [reply]
No, as above it is a configuration file setting.

**Comment by EmanueleCannizzaro** [comment]
good morning,

thnak you for linkchecker.
I have found no documentation on how to run checker within jupyter notebook.
Is it possible?

If so, what is the equivalent of linkchecker?
